runtime/include/FuncArgMacros.h

```diff
@@ -67,6 +67,8 @@
 #define F90_MATMUL(s) pg_mm_##s##_
 
 #else
+#endif
+#else
 #define ENTF90IO(UC, LC) f90io_##LC
 #define ENTF90(UC, LC) f90_##LC
```

Comment:
That's where the `f90_allocated` is coming from!

Reply:
Missing symbols should be here: https://github.com/flang-compiler/flang/blob/a9ccdd766d5e606a1dc073fa3b106e52bab5dc64/runtime/flang/allo.c